<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="WebRiskTests.WebRiskBasisTestRisiko" tests="1" time="31.452" ignored="0" failures="0" timestamp="30 MÃ¤r 2020 07:15:07 GMT" skipped="0" hostname="Laptop-BHDR" errors="1">
  <testcase name="BasisPHV" time="31.452" classname="WebRiskTests.WebRiskBasisTestRisiko">
    <error type="java.io.FileNotFoundException" message="D:\IR-WebRisk-Testdaten\Risiko\1-Risisko-Ergebnisseite.png (Das System kann den angegebenen Pfad nicht finden)">
      <![CDATA[java.io.FileNotFoundException: D:\IR-WebRisk-Testdaten\Risiko\1-Risisko-Ergebnisseite.png (Das System kann den angegebenen Pfad nicht finden)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(Unknown Source)
	at java.io.FileOutputStream.<init>(Unknown Source)
	at com.google.common.io.Files$FileByteSink.openStream(Files.java:192)
	at com.google.common.io.Files$FileByteSink.openStream(Files.java:180)
	at com.google.common.io.ByteSource.copyTo(ByteSource.java:268)
	at com.google.common.io.Files.copy(Files.java:319)
	at WebRiskTests.WebRiskBasisTestRisiko.BasisPHV(WebRiskBasisTestRisiko.java:183)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- BasisPHV -->
</testsuite> <!-- WebRiskTests.WebRiskBasisTestRisiko -->
